### PR TITLE
Adding the database connectivity code to the develop branch

### PR DESCRIPTION
**Pull request by Parth18Shah**

### CODE REVIEW

File: Models/patientDetails.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+class PatientDetails:
+    def __init__(self, email):
+        self.id = uuid.uuid4().hex
+        self.email = email
```
Comment (arpitMittalkenko) [inline Models/patientDetails.py:6]: This is a reduntant/duplicate field also present in user collection. Not needed in Patient collection

File: Models/ptDetails.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+class PTDetails:
+    def __init__(self, email):
+        self.id = uuid.uuid4().hex
+        self.email = email
```
Comment (arpitMittalkenko) [inline Models/ptDetails.py:6]: This is a reduntant/duplicate field also present in user collection. Not needed in PT collection

File: Models/ptDetails.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+class PTDetails:
```
Comment (arpitMittalkenko) [inline Models/ptDetails.py:3]: Change name to PhysicalTherapist

File: Models/patientDetails.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import uuid
+
+class PatientDetails:
```
Comment (arpitMittalkenko) [inline Models/patientDetails.py:3]: Change name to Patient

File: Models/user.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+class User:
+    def __init__(self, name, email, password, username=None, age=None, patient_id=None, ptdetails_id=None):
+        self.id = uuid.uuid4().hex
+        self.name = name
+        self.email = email
+        self.password = password
+        self.username = username
+        self.age = age
+        self.patient_id = patient_id
+        self.ptdetails_id = ptdetails_id
```
Comment (arpitMittalkenko) [inline Models/user.py:12]: change to physical_therapist_id

File: Utils/Utils.py [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+from passlib.hash import pbkdf2_sha256 as sha256
+
+client = MongoClient(Config.MONGO_DB_URL)
+db = client.kenko
+
+class Utils: 
+
+    @staticmethod
+    def getMongoDbClient():
+        client = MongoClient(Config.MONGO_DB_URL)
```
Comment (arpitMittalkenko) [inline Utils/Utils.py:12]: Do not create new Client everytime function is called. Create one client above (as done in signup branch) and return that

File: Utils/Utils.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+        return client
+    
+    @staticmethod
+    def getMongoDb():
+        client = Utils.getMongoDbClient()
+        db = client.kenko
```
Comment (arpitMittalkenko) [inline Utils/Utils.py:18]: Same as above

File: app.py [hunk 8]
```diff
@@ -1,5 +1,13 @@
 from flask import Flask
+import dotenv
+from dotenv import load_dotenv
+import os
+from routes import user_bp
+
```
Comment (arpitMittalkenko) [inline app.py:6]: remove un necessary imports.

File: dao/UserDao.py [hunk 9]
```diff
@@ -1,8 +1,28 @@
-import Utils.MongoDBConnection
-
-mongo_client = Utils.MongoDBConnection.getClient()
+import Utils.Utils as Utils
+# from Models.user import User
+# from Models.patientDetails import PatientDetails
+# from models.ptdetails import PTDetails
+db = Utils.Utils.getMongoDb()
```
Comment (arpitMittalkenko) [inline dao/UserDao.py:5]: import it as 
from Utils.Utils import Utils
db = Utils.getMongoDb()

File: facades/UserFacade.py [hunk 10]
```diff
@@ -0,0 +1,67 @@
+from Utils.Utils import Utils
+from services.UserService import UserService
+from Models.user import User
+from Models.patientDetails import PatientDetails
+from Models.ptDetails import PTDetails
+
+class UserFacade():
```
Comment (arpitMittalkenko) [inline facades/UserFacade.py:7]: Make all those functions as private which are only specific to this class and are not accessible outside.. in this case, only createNewUser will remain public.

File: Utils/Utils.py (hunk 6 above)
Comment (Parth18Shah) [reply Utils/Utils.py:12]: Corrected it.

File: Utils/Utils.py (hunk 7 above)
Comment (Parth18Shah) [reply Utils/Utils.py:18]: Corrected it.

File: app.py (hunk 8 above)
Comment (Parth18Shah) [reply app.py:6]: removed the dotenv and os import statements

File: facades/UserFacade.py (hunk 10 above)
Comment (Parth18Shah) [reply facades/UserFacade.py:7]: Understood, have updated the code accordingly.

File: Models/patientDetails.py (hunk 1 above)
Comment (Parth18Shah) [reply Models/patientDetails.py:6]: Got it will update the fields in my next commit.

File: Models/user.py (hunk 5 above)
Comment (Parth18Shah) [reply Models/user.py]: Updated it.

**Pull request merged**